crates/forge/src/lib.rs

```diff
@@ -54,13 +53,41 @@ static BUILTINS: Lazy<Vec<&str>> = Lazy::new(|| {
     ]
 });
 
+#[derive(Debug, PartialEq)]
+enum TestsToRun {
+    All,
+    Ignored,
+    NonIgnored,
+}
+
+impl TestsToRun {
+    fn from_flags(only_ignored: bool, include_ignored: bool) -> Self {
```

Comment:
What in case both `only_ignored` and `include_ignored` are true. Should we panic/return an error?

Reply:
They cannot be, guaranteed by clap
```rust
    #[arg(long, conflicts_with = "only_ignored")]
    include_ignored: bool,
```

Reply:
I understand this is configured in clap, but I think having an extra assert here could prevent some weird bugs in case logic elsewhere changes.

Reply:
Can add `assert` then